### ISSUE TITLE
Web Worker incompatibility

### ISSUE THREAD

**Issue by AsyncBanana** [kriszyp/msgpackr#35]
When I tried to use msgpackr within a web worker, I got the error
```
Uncaught ReferenceError: global is not defined
    at unpack.js:829
```
It seems like it would be easy enough to simply add another check and use `self` instead of global if both window and global were undefined.

**Comment by kriszyp** [comment]
Sounds like a good idea, I'll add that.

**Comment by kriszyp** [comment]
Maybe I could actually change `window` to `self`.

**Comment by AsyncBanana** [comment]
Thanks! Is there any way to use this since the NPM package has not gotten a new release?

**Comment by kriszyp** [comment]
Just published in v1.4.2.

**Comment by AsyncBanana** [comment]
It works now, thanks!
